x-pack/test/reporting_api_integration/reporting_and_security/usage.ts

```diff
@@ -7,9 +7,18 @@
 
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../ftr_provider_context';
-import * as GenerationUrls from '../services/generation_urls';
 import { ReportingUsageStats } from '../services/usage';
 
+// These all have the domain name portion stripped out. The api infrastructure assumes it when we post to it anyhow.
```

Comment:
Review note: Now that the shim is removed, this URLs are only used in this file, so I removed the entire GenerationUrls service to inline the URLs in this file.